pymc3/distributions/timeseries.py

```diff
@@ -309,8 +311,30 @@ def _random(self, sigma, mu, size, sample_shape):
         else:
             axis = 0
         rv = stats.norm(mu, sigma)
-        data = rv.rvs(size).cumsum(axis=axis)
-        data = data - data[0]  # TODO: this should be a draw from `init`, if available
+        if size is None:
```

Comment:
Can you add a small comment above this line, explaining why we have to do all this shape-handling now and writing down the link to the issue you opened on scipy?